.github/CODEOWNERS

```diff
@@ -1,2 +1,12 @@
 # add ospology-book-reviewers team to pull request that changes any file within the english book content
-/ospo-book/content/en/*.md     @todogroup/book-reviewers
+/ospo-book/content/en/*.md           @todogroup/book-reviewers
+
+# add ospology-book-infra to pull requests regarding spellcheck infra
+/.vale.ini                           @todogroup/ospology-book-infra
+/.vale.tmpl                          @todogroup/ospology-book-infra
+/.vale                               @todogroup/ospology-book-infra
+/.github/workflows/review-docs.yaml  @todogroup/ospology-book-infra
+
+# secure the repository
+/CODEOWNERS                          @anajsana
+/.github/settings.yml                @anajsana
```

Comment:
At first I added the [`@todogroup/awesome-ospo-admins`](https://github.com/orgs/todogroup/teams/awesome-ospo-admins), but they are not configured on the repository as collaborator, so unfortunately that generates an error. Might be a good idea to add an 'admin like' team and configure that team here.

Reply:
Who can do this, should we ask @anajsana ?

Reply:
Yeah, I guess Ana can do this.
I think it would be good if any change to the settings.yml and CODEOWNERS is reviewed by Ana (or a group where Ana is part of).

Reply:
I think @todogroup/steering-committee should be listed is /CODEOWNERS  and /.github/settings.yml  (cc @justaugustus )

Reply:
@anajsana, I missed this comment. I added a PR for that: https://github.com/todogroup/ospology/pull/568